web_src/js/features/repo-issue.js

```diff
@@ -680,3 +680,10 @@ export function initIssueTemplateCommentEditors($commentForm) {
     initCombo($(el));
   }
 }
+
+export function initArchivedLabelHandler() {
+  if (!document.querySelector('.archived-label-hint')) return;
```

Comment:
Why are we selecting for this hint? Seems brittle. Can the below loop stay on its own?

Reply:
I will refactor this naming a little bit.